src/zenml/zen_server/routers/users_endpoints.py

```diff
@@ -66,7 +70,7 @@
 current_user_router = APIRouter(
     prefix=API + VERSION_1,
     tags=["users"],
-    dependencies=[Depends(authorize)],
+    dependencies=[Security(authorize, scopes=["read"])],
```

Comment:
Even users with no permissions should be able to fetch information about their account.

Reply:
Good point